src/mpool.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+#pragma once
+
+#include <stdlib.h>
+#include <string.h>
+#if !defined(_WIN32)
```

Comment:
No, implementation specific statements should no appear in this header! Keep it simple and stupid.